qa/tasks/cephadm.py

```diff
@@ -304,8 +304,10 @@ def first_in_ceph_log(pattern, excludes):
             """
             args = [
                 'sudo',
-                'egrep', pattern,
-                '/var/log/ceph/{fsid}/ceph.log'.format(
+                'egrep', '-r',
+                '--include=ceph-mon.*.log',
```

Comment:
This part might get resolved through https://github.com/ceph/ceph/pull/54312, so I would suggest dropping "Set correct log location for cephadm task" commit (at least for now).  Coercing cephadm into giving us an actual cluster log file with a well-defined name seems much preferred to grepping raw monitor logs.